### PR TITLE
fix_wrong_word_spelling Update PSR-6-cache.md

### PR DESCRIPTION
**Pull request by phpseasa**
i think the psr-6 has a wrong word spelling ,  the word 'it' should be 'is' , pls check my pr and thx

### CODE REVIEW

File: accepted/PSR-6-cache.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ below with whole-second granularity.
 when that item is stored and it is considered stale. The TTL is normally defined
 by an integer representing time in seconds, or a DateInterval object.
 
-*    **Expiration** - The actual time when an item is set to go stale. This it
+*    **Expiration** - The actual time when an item is set to go stale. This is
```
Comment (Jean85) [inline accepted/PSR-6-cache.md]: Did you add a `&nbsp;` here?

Comment (phpseasa) [inline accepted/PSR-6-cache.md]: no ,  i mean the word 'it' should be 'is' . 'This it' should be 'This is' .

Comment (phpseasa) [inline accepted/PSR-6-cache.md]: '&nbsp' works fine ,no modify on that

**Comment by michaelcullum** [comment]
Thanks!

**Pull request merged**